README.md

```diff
@@ -157,6 +162,25 @@ Echo.join('channel3');
 Echo.leaveChannel("presence-channel3")
 ```
 
+**2. Error handling**
+- Please note that the [Ably laravel-echo client](https://github.com/ably-forks/laravel-echo) emits [Ably specific error codes](https://github.com/ably/ably-common/blob/main/protocol/errors.json) instead of [Pusher error codes](https://pusher.com/docs/channels/library_auth_reference/pusher-websockets-protocol/#error-codes).
+- Ably emitted errors are descriptive and easy to understand, so it's more effective to take a corrective action ( Pusher errors mostly emitted as integer error codes ).
+- Ably errors are provided as an [ErrorInfo object](https://ably.com/docs/api/realtime-sdk/types#error-info) with full error context.
+- If you are interacting with pusher errors in your project, be sure to update your code accordingly.
+i.e. update from [pusher error codes](https://pusher.com/docs/channels/library_auth_reference/pusher-websockets-protocol/#error-codes) to [ably error codes](https://github.com/ably/ably-common/blob/main/protocol/errors.json).
+
+```js
+    channel.error(error => {
+        if (error && error.code === 40142) { // ably token expired
+            console.error(error);
+            // take corrective action on UI
+        }
+    })
+```
+**Note :**
+- AblyPresenceChannel -> `here` method gets called everytime client **joins, updates or leaves** the channel unlike pusher client (gets called only for the first time).
```

Comment:
```suggestion
- AblyPresenceChannel -> `here` method gets called every time a client **joins, updates or leaves** the channel, whereas when using Pusher this is only called when a client enters.
```

Reply:
It's not when `client` enters. It's called only once in the whole lifetime for the first client entering the channel.

Reply:
updated!